src/registered/formatters.js

```diff
@@ -348,6 +348,7 @@ function registerDefaults(fragments) {
 
 
   var urlExp = /(^|\s|\()((?:https?|ftp):\/\/[\-A-Z0-9+\u0026@#\/%?=()~_|!:,.;]*[\-A-Z0-9+\u0026@#\/%=~(_|])/gi;
+  var wwwExp = /(^|[^\/])(www\.[\S]+(\b|$))/gim;
```

Comment:
Ooh, what about `/(^|[^\/])([\S]+\.[\S]+\.\w{2,}(\b|$))/gim` to match subdomain.domain.com?